src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+## This is a helper script for non-containerized local build and test execution.
+## It downloads and uses the daily SDK which contains the compatible AspNetCore bits.
+## Usage:
+## ./build-local.sh [StressConfiguration] [LibrariesConfiguration]
+
+version=7.0
+repo_root=$(git rev-parse --show-toplevel)
+daily_dotnet_root=./.dotnet-daily
+
+stress_configuration="Release"
+if [ "$1" != "" ]
+then
```

Comment:
```suggestion
if [ -z "$1" ]; then
```

Reply:
I need to change a bunch of scripts though, I used it consistently in this way.